src/App.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
 
+import Login from "./components/Login";
+
 function App() {
```

Comment:
Por cuestiones de estilo, es mejor usar funciones flechas en vez de funciones normales. Como por lo general `App` es un componente presentacional (no tiene lógica), en vez de utilizar llaves directamente ponemos el `jsx` luego de la flexha

 ```suggestion
const App = () => 
 <div className="App">
```